Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m

```diff
@@ -1168,50 +1208,41 @@ - (void)prepareForReuse
 {
     [super prepareForReuse];
 
-    [NSLayoutConstraint deactivateConstraints:self.payloadConstraints];
-    self.payloadConstraints = nil;
-    [NSLayoutConstraint deactivateConstraints:self.contentConstraints];
-    self.contentConstraints = [NSMutableArray new];
-    [NSLayoutConstraint deactivateConstraints:self.dateHeaderConstraints];
-    self.dateHeaderConstraints = nil;
-    [NSLayoutConstraint deactivateConstraints:self.footerConstraints];
-    self.footerConstraints = nil;
+    [NSLayoutConstraint deactivateConstraints:self.viewConstraints];
+    self.viewConstraints = [NSMutableArray new];
 
     self.dateHeaderLabel.text = nil;
     self.dateHeaderLabel.hidden = YES;
-    self.textView.text = nil;
-    self.textView.hidden = YES;
-    self.textView.dataDetectorTypes = UIDataDetectorTypeNone;
+    [self.bodyTextViewCached removeFromSuperview];
```

Comment:
What are you trying to indicate with the term `cached` in this variable? There is no `uncached` bodyTextView is there? 

Is it that we're only creating the textView during init, and not nilling/rebuilding every re-use cycle for perf?

Reply:
You're right - the name is confusing.  Removed `Cached`.